easybuild/easyconfigs/b/BLAT/BLAT-3.5-foss-2019a.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 The Cyprus Institute
+# Authors::   Andreas Panteli <a.panteli@cyi.ac.cy>, Thekla Loizou <t.loizou@cyi.ac.cy>
+# License::   MIT/GPL
+#
+##
+
+name = 'BLAT'
+version = '3.5'
+
+homepage = 'http://genome.ucsc.edu/FAQ/FAQblat.html'
+description = """BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or
+ more."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
I think we can move this down to the `GCC` toolchain?

Reply:
Good idea. I moved it too `GCCcore` though, as BLAT could still be used as a dependency in the `intel` toolchain.

Reply:
@lexming Typically, for things that seem potentially performance critical we typically prefer them at the `GCC` level, so that we can make a `iccifort` which, often, is faster.

Typically, this is half-guessing, but, icc is often really good. In some cases, I have actually decided to benchmark the software, and, if gcc is at least as good, then I think just a GCCcore version is preferable. But, if you can't benchmark it, I think it's best to stick to GCC, and leave it open to add a iccifort version later on.

Reply:
Ok, no problem. Updated using the `GCC` toolchain.